config/main.py

```diff
@@ -3252,6 +3253,9 @@ def reload(ctx, no_dynamic_buffer, no_delay, dry_run, json_data, ports, verbose)
     if buffer_model_updated:
         print("Buffer calculation model updated, restarting swss is required to take effect")
 
+    if not status:
+        sys.exit(1)
```

Comment:
It will always exit(1) when it is --no-delay.

Reply:
Ah, yes, I created a new patch to fix it.